File: src/main/java/com/deflatedpickle/heartdrops/items/ItemBase.java
package com.deflatedpickle.heartdrops.items;

import com.deflatedpickle.heartdrops.Reference;
import net.minecraft.creativetab.CreativeTabs;
import vazkii.arl.item.ItemMod;

public class ItemBase extends ItemMod {
    public ItemBase(String name) {
        super(name);
        setMaxStackSize(1);
        setCreativeTab(CreativeTabs.MISC);
    }

    @Override
    public String getModNamespace() {
        return Reference.MOD_ID;
    }
}

File: src/main/scala/com/deflatedpickle/heartdrops/items/ItemBase.scala
package com.deflatedpickle.heartdrops.items

import com.deflatedpickle.heartdrops.Reference
import net.minecraft.creativetab.CreativeTabs
import vazkii.arl.item.ItemMod

class ItemBase(val name: String) extends ItemMod(name) {
  setMaxStackSize(1)
  setCreativeTab(CreativeTabs.MISC)

  override def getModNamespace: String = Reference.MOD_ID
}
